### DISCUSSION TITLE
We got cute preview image!

### DISCUSSION THREAD

**Discussion by abiriadev** [abiriadev/seia#4] [Show and tell]
@nemorize has created an adorable image for GitHub preview!

![image](https://github.com/abiriadev/seia/assets/72962900/fbc98606-db29-4e56-8fb4-6b5e19549e18)

Let's give him a big round of applause!

**Comment by nemorize** [comment]
I've attached the original files for the preview image for you to use if you need to make any further modifications.
Two files are in the archive: an .afphoto file created using Affinity Photo 2 (https://affinity.serif.com/en-us/photo/), and a PSD conversion of that file.

I can't guarantee the quality of the PSD file conversion, so I recommend using Affinity Photo.

Take care.

[abiriadev-seia-preview-images.zip](https://github.com/user-attachments/files/15593521/abiriadev-seia-preview-images.zip)
`SHA2-256(abiriadev-seia-preview-images.zip)= b0df1db1bc01f845ff3cdacb5507ee5bf56eb78261f4f72fcceebe9b03b0060e`

**Reply by abiriadev** [reply]
Thanks a lot!! :heart:
